indico/modules/events/editing/client/js/components/FileTypeManager.jsx

```diff
@@ -40,24 +102,106 @@ export default function FileTypeManager({eventId}) {
         <Segment key={fileType.id} styleName="filetype-segment">
           <div styleName="filetype">
             <div>
-              <h3>{fileType.name}</h3>
+              <h3>
+                <span styleName="name">{fileType.name}</span>
+                {fileType.required && (
+                  <Icon
+                    size="small"
+                    color="grey"
+                    name="asterisk"
+                    title={Translate.string('File required')}
+                  />
+                )}
+                {fileType.allowMultipleFiles && (
+                  <Icon
+                    size="small"
+                    color="grey"
+                    name="copy outline"
+                    title={Translate.string('Multiple files allowed')}
+                  />
+                )}
+                {fileType.publishable && (
+                  <Icon
+                    size="small"
+                    color="grey"
+                    name="eye"
+                    title={Translate.string('File publishable')}
+                  />
+                )}
+              </h3>
               <ul styleName="file-extensions">
                 {fileType.extensions.length !== 0
                   ? fileType.extensions.map(ext => <li key={ext}>{ext}</li>)
                   : Translate.string('(no extension restrictions)')}
               </ul>
             </div>
             <div styleName="actions">
-              <Icon color="blue" name="pencil" />
-              <Icon color="red" name="trash" />
+              <Icon
+                color="blue"
+                name="pencil"
+                onClick={() => dispatch({type: 'EDIT_FILE_TYPE', fileType})}
+              />
+              <Popup
+                on="hover"
+                disabled={!fileType.isUsed}
+                position="right center"
+                content={Translate.string('This type has files attached')}
```

Comment:
I would change it to something like: 'Not possible to delete: type has files attached'

Reply:
not sure if that's needed - with a grayed-out delete button it's quite obvious. and it looks a bit ugly if it gets wrapped on a second line.

Reply:
For us it is obvious, but I am always in for being more explicit, because some users get confused easily. Having said that, we can leave it up to @ParthS007 to decide